lib/kredis/types/list.rb

```diff
@@ -16,12 +17,20 @@ def remove(*elements)
     types_to_strings(elements, typed).each { |element| lrem 0, element }
```

Comment:
Removal doesn't freshen expiry, though?

Feeling like there will be lots of gaps in this behavior. Should be moved to a declaration on the underlying proxied operations?